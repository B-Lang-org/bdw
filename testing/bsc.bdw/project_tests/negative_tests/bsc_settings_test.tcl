
# Opens the specified project
handlers::open_project project.bspec

# Calls the set_compilation_type without arguments
handlers::set_compilation_type 

# Calls the set_compilation_type with a wrong argument
handlers::set_compilation_type wrong_comp_type

# Sets the bsc type to bluesim
handlers::set_bsc_options

# Sets the bsc type to bluesim
handlers::set_bsc_options -bluesimmm

# Sets the bsc type to bluesim
handlers::set_bsc_options -bluesim -opt -v

# Sets the compilation result directories
handlers::set_compilation_results_location 

# Sets the compilation result directories
handlers::set_compilation_results_location -vdir 

# Sets the compilation result directories
handlers::set_compilation_results_location -vdir mult4 -vdir mult4

# Sets the compilation result directories
handlers::set_compilation_results_location -vdir mult4 -bdir

# Closes the project
handlers::close_project

# Exits the workstation
exit
