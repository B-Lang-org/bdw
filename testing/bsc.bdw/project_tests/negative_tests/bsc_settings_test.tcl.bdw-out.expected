The compilation type is not specified
The compilation type should be either make or bsc.
There is no option specified
Unknown option: -bluesimmm
Unknown option: -opt
There are no arguments specified
Argument -vdir expects an argument none found
The option -vdir has already been specified
Argument -bdir expects an argument none found
